packages/dappmanager/src/modules/chains/getChainDriverName.ts

```diff
@@ -18,9 +18,5 @@ const knownChains: { [dnpName: string]: ChainDriver } = {
   "bitcoin.dnp.dappnode.eth": "bitcoin",
   "monero.dnp.dappnode.eth": "monero",
   "prysm.dnp.dappnode.eth": "ethereum2-beacon-chain-prysm",
-  "prysm-prater.dnp.dappnode.eth": "ethereum2-beacon-chain-prysm",
-  "teku.dnp.dappnode.eth": "ethereum2",
-  "teku-prater.dnp.dappnode.eth": "ethereum2",
-  "lighthouse.dnp.dappnode.eth": "ethereum2",
-  "lighthouse-prater.dnp.dappnode.eth": "ethereum2"
+  "prysm-prater.dnp.dappnode.eth": "ethereum2-beacon-chain-prysm"
```

Comment:
`"prysm-prater.dnp.dappnode.eth": "ethereum-beacon-chain"`

Let's not promote the legacy name

Reply:
Also, is this necessary anymore? What the last Prysm version that does not include this in the manifest?

Reply:
It will be renamed in the future, PRs to implement such change: 
- SDK  https://github.com/dappnode/DAppNodeSDK/issues/190
- Prysm  https://github.com/dappnode/DAppNodePackage-prysm/pull/62
- Prysm-prater   https://github.com/dappnode/DAppNodePackage-prysm-prater/pull/35
- Docs   https://github.com/dappnode/DAppNodeDocs/issues/135

For the moment let's keep the name `ethereum2-beacon-chain-prysm` to avoid breaking changes